uiowa/viewbook-law
# College of Law Viewbook: initial customizations

Relates to https://github.com/uiowa/uiowa/issues/3289

# How to test

- Install

`yarn`

- Start the local server
- The following command creates and runs a temporary server to view the site locally.

`yarn dev`

- The command will output the server URL to visit in your browser, ex. http://localhost:3000.
- Test the build locally

- The following command creates the build.

`yarn build`

- The following command creates and runs a temporary python server to view the site in dist locally.

`cd dist`

`python3 -m  http.server 8000 `

The site should look like the screenshot below (minus the weird scrollbar)

![Screenshot_2021-06-24 VueJS Site Template The University of Iowa](https://user-images.githubusercontent.com/1036433/123288234-fbf19180-d4d4-11eb-8649-76d505ac7f38.jpg)
